Sources/Java2SwiftLib/JavaClassTranslator.swift

```diff
@@ -104,12 +104,25 @@ struct JavaClassTranslator {
     }
 
     let genericParameters = javaTypeParameters.map { param in
-      "\(param.getName()): AnyJavaObject"
+      let name = generateGenericParameterName(param.getName())
+      translator.usedGenericParameterNames.insert(param.getName())
+      return "\(name): AnyJavaObject"
     }
 
     return "<\(genericParameters.joined(separator: ", "))>"
   }
 
+  /// If the name already exists, append a 1 to it.
+  /// For example, if name = "T" and it already exists, the result will be "T1".
+  private func generateGenericParameterName(_ baseName: String) -> String {
+    var name = baseName
+    let index = 1
+    if translator.usedGenericParameterNames.contains(name) {
+      name = "\(baseName)\(index)"
+    }
+    return name
```

Comment:
This would probably have to be more contextual "in this context (of a function or type) there already was a T used so make it a T1". Especially with types nested in other types where there's a T outside and then another T etc...